deploy/ray-operator/utils/util.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"strings"
+)
+
+// The function isCreated returns true if pod has been created and is maintained by the API server
+func IsCreated(pod *corev1.Pod) bool {
+	return pod.Status.Phase != ""
+}
+
+func Before(value string, a string) string {
+	// Get substring before a string.
+	pos := strings.Index(value, a)
+	if pos == -1 {
+		return ""
+	}
+	return value[0:pos]
+}
+
+// FormatInt returns the string representation of i in the given base,
+// for 2 <= base <= 36. The result uses the lower-case letters 'a' to 'z'
+// for digit values >= 10.
+func FormatInt32(n int32) string {
```

Comment:
```suggestion
func formatInt32(n int32) string {
```

Reply:
In Go, capital letter means the function can be accessed by other package, lower-case means private can only be accessed in the same package